starlite/middleware/exceptions/middleware.py

```diff
@@ -47,6 +47,8 @@ async def __call__(self, scope: "Scope", receive: "Receive", send: "Send") -> No
             await self.app(scope, receive, send)
         except Exception as e:  # pylint: disable=broad-except
             starlite_app = scope["app"]
+            if self.debug and (logger := starlite_app.logger):
```

Comment:
```suggestion
            if self.debug and (logger := starlite_app.get_logge()):
```

no?

Reply:
Perhaps, but it broke my test 🙃 looking into why tonight hopefully

Reply:
Maybe because the function isn’t called `get_logge` but `get_logger`? 😉

Reply:
i fixed that as soon as i committed the suggestion and realized 😛

Reply:
If this change breaks the test, it most likely is because we are patching `app.logger` with response of calling the `get_logger` fixture in the test, whereas for this to work, we'd have to patch the `app.get_logger` attribute with the get_logger fixture itself.

In the starlite constructor, we both assign the get_logger callable and a logger instance to the application instance:

```python
        if self.logging_config:
            self.get_logger = self.logging_config.configure()
            self.logger = self.get_logger("starlite")
```
...and it wasn't obvious to me which one we should be using in this case either.